ajax/libs/can/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "can",
+  "title": "CanJS",
+  "description": "MIT-licensed, client-side, JavaScript framework that makes building rich web applications easy.",
+  "filename": "can.js",
+  "keywords": [
+    "CanJS",
+    "DoneJS"
+  ],
+  "author": {
+    "name": "Bitovi",
+    "email": "contact@bitovi.com",
+    "web": "https://bitovi.com/"
+  },
+  "homepage": "https://canjs.com/",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:canjs/canjs.git",
+    "github": "https://github.com/canjs/canjs"
+  },
+  "license": "MIT",
+  "npmName": "can",
```

Comment:
There are 138 releases on GitHub but 111 on NPM. It would be better to use git auto-update config instead. How do you think @christopherjbaker? You can take a look at [autoupdate.md](https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md).